### DISCUSSION TITLE
cara push code yang yang hilang ketika lupa memasukkan perintah git add .

### DISCUSSION THREAD

**Discussion by falfikri** [imamsudarajat04/nongkrongkoding#27] [Q&A] [answered]
kemaren aku mau push eh taunya malah kelupaan nambah perintah git add . nah pas aku commit sama push jadi gabisa nah itu gimana tu memperbaiki masalahnya tu

**Comment by imamsudarajat04** [comment, marked as answer]
you can try this :
$ git pull origin master --allow-unrelated-histories
$ git push origin master
note master or main this your branch repo!
